[contacts] add isAvailableAsync method (#9640)
expo/expo: docs/pages/versions/unversioned/sdk/contacts.md

import * as Contacts from 'expo-contacts';
```

> <TableOfContentSection title='Methods' contents={['Contacts.isAvailableAsync', 'Contacts.requestPermissionsAsync', 'Contacts.getPermissionsAsync', 'Contacts.getContactsAsync(contactQuery)', 'Contacts.getContactByIdAsync(contactId, fields)', 'Contacts.addContactAsync(contact, containerId)', 'Contacts.updateContactAsync(contact)', 'Contacts.presentFormAsync(contactId, contact, formOptions)', 'Contacts.removeContactAsync(contactId)', 'Contacts.writeContactToFileAsync(contactQuery)']} />

Author: ```suggestion
<TableOfContentSection title='Methods' contents={['Contacts.isAvailableAsync()', 'Contacts.requestPermissionsAsync', 'Contacts.getPermissionsAsync', 'Contacts.getContactsAsync(contactQuery)', 'Contacts.getContactByIdAsync(contactId, fields)', 'Contacts.addContactAsync(contact, containerId)', 'Contacts.updateContactAsync(contact)', 'Contacts.presentFormAsync(contactId, contact, formOptions)', 'Contacts.removeContactAsync(contactId)', 'Contacts.writeContactToFileAsync(contactQuery)']} />
```

## Methods

### `Contacts.isAvailableAsync()`

> Returns whether the Contacts API is enabled on the platform or device. This does not consider user permissions, that must be checked separately.

Author: ```suggestion
Returns whether the Contacts API is enabled on the current device. This does not check the app permissions.
```

#### Returns

> Async `boolean`, indicating whether the Contacts API is available on this device. Currently this only returns `false` in the web browser.

Author: ```suggestion
Async `boolean`, indicating whether the Contacts API is available on the current device. Currently this resolves to `true` on iOS and Android only.
```
